doc/explanation/lxc_show_info.md

```diff
@@ -0,0 +1,49 @@
+(lxc-show-info)=
+# About `lxc` 'show' and 'info'
+For the entities managed by LXD, the `lxc` command provides a 'list' sub-command, and may provide 'show' and 'info' sub-commands.
+The purpose of the 'info' sub-command is to show current state information, and the purpose of the 'show' sub-command is to show configuration information and how the entity is used by other entities.
+
+For example, the `lxc network info` command shows IP address and traffic statistics;
+
+    Name: lxdbr0
+    MAC address: 00:16:3e:d3:ec:41
+    MTU: 1500
+    State: up
+
+    Ips:
+      inet    10.247.40.1
+      inet6   fd42:d389:f4a1:53d2::1
+      inet6   fe80::216:3eff:fed3:ec41
+
+    Network usage:
+      Bytes received: 127.66kB
+      Bytes sent: 15.54kB
+      Packets received: 1433
+      Packets sent: 175
+
+and the `lxc network show` command shows further information, and which entities are using the network:
```

Comment:
```suggestion
The `lxc network show` command, on the other hand, shows how the network is configured, and which entities are using the network:
```

Two comments here:
- You shouldn't continue a sentence if you have code blocks in the middle of it. This works okay in English, but it might cause problems if the documentation is ever translated into other languages where this is not possible.
- I suggest to add in "on the other hand" to make it a bit clearer what we're trying to say in this page - we want to show the difference between the two commands.